docker/docker-entrypoint.sh

```diff
@@ -8,14 +8,16 @@ set -e
 DB_WAIT_TIMEOUT=${DB_WAIT_TIMEOUT-3}
 MAX_DB_WAIT_TIME=${MAX_DB_WAIT_TIME-30}
 CUR_DB_WAIT_TIME=0
-while ! nautobot-server post_upgrade --no-invalidate-all 2>&1 && [ "${CUR_DB_WAIT_TIME}" -lt "${MAX_DB_WAIT_TIME}" ]; do
-  echo "⏳ Waiting on DB... (${CUR_DB_WAIT_TIME}s / ${MAX_DB_WAIT_TIME}s)"
-  sleep "${DB_WAIT_TIMEOUT}"
-  CUR_DB_WAIT_TIME=$(( CUR_DB_WAIT_TIME + DB_WAIT_TIMEOUT ))
-done
-if [ "${CUR_DB_WAIT_TIME}" -ge "${MAX_DB_WAIT_TIME}" ]; then
-  echo "❌ Waited ${MAX_DB_WAIT_TIME}s or more for the DB to become ready."
-  exit 1
+if [ ! "$NAUTOBOT_DOCKER_SKIP_INIT" ]; then
```

Comment:
Does it make sense for this to be a separate environment variable, or should we instead check for `$NAUTOBOT_MAINTENANCE_MODE` so that the user doesn't have to set two different variables to get the correct behavior??

Reply:
Good points, thanks. :-)